common/reactive-grpc-common/src/test/java/com/salesforce/reactivegrpc/common/StreamObserverAndPublisherTest.java

```diff
@@ -228,11 +228,11 @@ public void run() {
         }
     }
 
+    // TODO FIX THIS
     static class FussedTestSubscriber<T> extends TestSubscriber<T> {
```

Comment:
TestSubscriber no longer has initialFusionMode available and that functionality seems to not exist anymore inside TestSubscriber in RxJava3.